### PR TITLE
Fixed failure of networking tests.

### PR DESCRIPTION
**Pull request by yossisegev**
The default MTU on the pod network interface varies, depending
on the environment on which the cluster runs, and it made some
of the tests to fail.
Now the test covers all the possible MTU values.

### CODE REVIEW

**Comment by SchSeba** [comment]
@yossisegev please run the `make` command to make travis happy again :)

**Comment by yossisegev** [comment]
ci please test

File: tests/vmi_networking_test.go [hunk 1]
```diff
@@ -190,9 +188,25 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
...
-			Expect(strings.Contains(output, expectedMtuString)).To(BeTrue())
+
+			// The MTU of the Pod network varies, depending on the environment in use. We want to
+			// verify that the actual MTU is from the possible range, which is {1500, 1450}, minus
+			//  50 bytes for vxlan overhead.
+			possibleMtus := []int{1400, 1450}
+			mtuMatch := false
+			expectedMtuString := ""
+
+			for _, expectedMtu := range possibleMtus {
```
Comment (rmohr) [inline tests/vmi_networking_test.go]: I guess we could find that out once and don't do it every time, but the change is great.

**Pull request merged**